README.md

```diff
@@ -57,6 +56,11 @@ data:
     username=core
 ```
 
+#### Removing BYOH Windows instances
+BYOH instances that have been joined to the cluster as a node can be removed by deleting the instance's entry in the
```

Comment:
I'm unsure if "joined" is a term, but if it's not, maybe "BYOH instances that are attached to the cluster..."